src/content/docs/browser/new-relic-browser/troubleshooting/browser-data-doesnt-match-other-analytics-tools.mdx

```diff
@@ -24,9 +24,11 @@ Follow these troubleshooting steps for browser monitoring:
 
 1. Check whether browser monitoring is properly installed in all of your application's pages.
 2. If the JavaScript snippets are missing, or if users are unable to send data to New Relic, follow the steps for [troubleshooting browser monitoring installation](/docs/browser/new-relic-browser/installation-configuration/troubleshooting-browser-monitoring-installation).
-3. If you have a single-page style application and are expecting to see your route changes as views, consider using [browser SPA monitoring](/docs/browser/single-page-app-monitoring/get-started/welcome-single-page-app-monitoring), which provides an integrated view of initial page loads and route changes.
+3. If you have a single-page style application and are expecting to see your route changes as views, consider using [browser SPA monitoring](/docs/browser/single-page-app-monitoring/get-started/welcome-single-page-app-monitoring), which provides an integrated view of both initial page loads and route changes.
 
-If browser monitoring is working properly, you may be seeing different results because New Relic collects and presents information differently from other analytics tools. For more information about how we collects data, see:
+If browser monitoring is working properly, you may be seeing different results because New Relic collects and presents information differently from other analytics tools. For instance, `JavascriptErrors` and `AjaxRequest` are individually monitored and their data depends on when our agent observes them and what we can or decide to capture. For `BrowserInteraction`, we use our own custom instrumentation. And for vitals and metrics under `PageViewTiming` and `PageView`, the web-vitals library owned by Chrome team is the source of that data.
```

Comment:
```suggestion
If browser monitoring is working properly, you may be seeing different results because New Relic collects and presents information differently from other analytics tools. For instance, `JavascriptErrors` and `AjaxRequest` are individually monitored and their data depends on when our agent observes them and what we can or decide to capture. For `BrowserInteraction`, we use our own custom instrumentation, and for vitals and metrics under `PageViewTiming` and `PageView`, the web-vitals library owned by Chrome team is the source of that data.
```